internal/observability/graph.go

```diff
@@ -230,6 +251,141 @@ func newEdge(logger *zap.Logger, meter *metrics.Scope, req *transport.Request, d
 	if err != nil {
 		logger.Error("Failed to create server failure latency distribution.", zap.Error(err))
 	}
+
+	// streaming requests
+	var streaming *streamEdge
+	if rpcType == transport.Streaming {
+		// sends
+		sends, err := meter.Counter(metrics.Spec{
+			Name:      "stream_sends",
+			Help:      "Total number of streaming messages sent.",
+			ConstTags: tags,
+		})
+		if err != nil {
+			logger.Error("Failed to create streaming sends counter.", zap.Error(err))
```

Comment:
what could cause us to fail to create these? if it's something that shouldn't ever happen at runtime, should this be a `DPanic`?

Reply:
There's a global registry so all metric names must be unique within the same process. `DPanic` probably better fits here.